test/Bin/GameTest/GameTests_1500.cpp

```diff
@@ -160,3 +160,10 @@ GAME_TEST(Issues, Issue1597) {
     EXPECT_NE(attrEnchantmentsNum, 0);
     EXPECT_NE(specialEnchantmentsNum, 0);
 }
+
+// 1600
+GAME_TEST(Issues, Issue1657) {
+    // Party can be placed at wrong position after canceling indoor transfer and changing map after it
+    test.playTraceFromTestData("issue_1657.mm7", "issue_1657.json");
+    EXPECT_EQ(pParty->pos.toInt(), Vec3i(12552, 800, 193)); // party is back at new game start position
```

Comment:
Let's also check that:
1. Indoor travel screen was opened.
2. Pin down the current map tape. Should be two values there, right?